README.md

```diff
@@ -1,3 +1,30 @@
-# Greek Digital Community webpage
+# Webpage for Greek Digital Community
 
-Playing with bootstrap and github pages. Ανεβάζω και κανα hugo αν ντε και καλά θέλουμε εκείνο. Κανένα πρόβλημα πάντως. Ψήνομαι να παίξω και μ'εκείνο. Προς το παρόν το bootstrap δείχνει πολύ άδειο έχοντας μόνο τη φωτό με τις λάμπες.
+## Introduction
+
+This repository contains the main website for the Greek Digital Community in 
+Slack. You can visit it here: [http://greekdigitalcommunity.com](http://greekdigitalcommunity.com).
+
+The theme is using bootstrap and is static html. To contribute please follow the
+instructions below.
+
+## Create a local copy
+First of all, please fork the repository.  
```

Comment:
lets put some thought on that

Reply:
Εγώ λέω να την παρκάρουμε την κουβέντα και να πάρουμε και από τους άλλους γνώμες. Έτσι είναι και για τα invites που έφτιαξε ο Θάνος οπότε θα πρέπει να τα δούμε και τα 2.